### PR TITLE
[C-1615] Fix android fullscreen modals

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes issue where modals are not full screen on android, due to their positioning in navigation hierarchy. moving to the top solves the issue, at the compromise of encapsulation.

### CODE REVIEW

File: packages/mobile/android/app/src/main/res/values-night/colors.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<resources>
```
Comment (sliptype) [inline packages/mobile/android/app/src/main/res/values-night/colors.xml:1]: Why is this file needed?

**Review by sliptype — APPROVED** [review]

File: packages/mobile/android/app/src/main/res/values-night/colors.xml (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/android/app/src/main/res/values-night/colors.xml:1]: good catch, its meant for another pr, which is general android mobile-upload fixes. this is for the date-picker interestingly enough
